linux/.gitignore

```diff
@@ -80,4 +80,6 @@ keyman-config/build
 keyman-config/debian/man
 keyman-config/experiments/*.txt
 keyman-config/keyman_config/version.py
+keyman-config/locale/*/LC_MESSAGES/
+*.po
```

Comment:
Is the plan to eventually check in the .po files from crowdin into the repo, where we would't want to ignore `*.po`?

Reply:
Ah, yes, you're right. I had the same thought this morning...
Done.